Dockerfile

```diff
@@ -1,76 +0,0 @@
-# FROM java:8
-FROM alpine:3.7
-
-ENV SONAR_VERSION=6.1 \
-    SONARQUBE_HOME=/opt/sonarqube \
-    SONAR_RUNNER_VERSION=2.4 \
-    SONAR_RUNNER_HOME=/opt/sonar-runner \
-    # Database configuration
-    # Defaults to using H2
-    SONARQUBE_JDBC_USERNAME=sonar \
-    SONARQUBE_JDBC_PASSWORD=sonar \
-    SONARQUBE_JDBC_URL=
-
-ENV JAVA_HOME="/usr/lib/jvm/java-1.8-openjdk"
-
-# Http port
-EXPOSE 9000
```

Comment:
I think we can remove `EXPOSE` as it is used only for local docker networking. 

https://we-are.bookmyshow.com/understanding-expose-in-dockerfile-266938b6a33d

Reply:
Good call